Listenbrainz.xcodeproj/project.pbxproj

```diff
@@ -728,7 +740,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0.0;
-				PRODUCT_BUNDLE_IDENTIFIER = org.listenbrainz.ios;
+				PRODUCT_BUNDLE_IDENTIFIER = org.listenbrainz.ios23;
```

Comment:
Don't commit this change. Keep this locally for your development @gauravbhardwaj7 . Maybe just put this file in the .gitignore file

Reply:
It's pbxproj file, It contains the reference paths too, so I was not finding a way to change bundle identifier in particular.